content/docs/operations/debug.md

```diff
@@ -5,46 +5,41 @@ weight: 15
 
-1. Check the {{< reuse "docs/snippets/product-name.md" >}} installation. You can do that by using the `{{< reuse "docs/snippets/cli-name.md" >}} check` [command](/docs/reference/cli/glooctl_check/) that quickly checks the health of {{< reuse "docs/snippets/product-name.md" >}} deployments, pods, and custom resources, and verifies Gloo resource configuration. Any issues that are found are reported back in the CLI output. 
+1. Make sure that the {{< reuse "docs/snippets/product-name.md" >}} control plane and gateway proxies are running.
+   
+   ```shell
+   kubectl get pods -n {{< reuse "docs/snippets/ns-system.md" >}}
+   ```
+   <!-- TODO: CLI You can do that by using the `{{< reuse "docs/snippets/cli-name.md" >}} check` [command](/docs/reference/cli/glooctl_check/) that quickly checks the health of {{< reuse "docs/snippets/product-name.md" >}} deployments, pods, and custom resources, and verifies Gloo resource configuration. Any issues that are found are reported back in the CLI output. 
    ```sh
    {{< reuse "docs/snippets/cli-name.md" >}} check
    ```
    
    Example output for a misconfigured VirtualHostOption:
    ```console
-   Found rejected VirtualHostOption by 'gloo-system': gloo-system jwt (Reason: 2 errors occurred:
+   Found rejected VirtualHostOption by '{{< reuse "docs/snippets/ns-system.md" >}}': {{< reuse "docs/snippets/ns-system.md" >}} jwt (Reason: 2 errors occurred:
 	* invalid virtual host [http~bookinfo_example] while processing plugin enterprise_warning: Could not load configuration for the following Enterprise features: [jwt]
    ```
-   
-2. Get the details for the failed resource. For example to get the details of a VirtualHostOption, you can use the following command. 
+   -->
+2. Get the details for any failed resource. For example to get the details of an HTTPRoute, you can use the following command. 
    ```sh
-   kubectl get virtualhostoption <name> -n <namespace>
+   kubectl get httproute <name> -n <namespace>
    ```
    
-3. If the resources seem to be ok, you can check the configuration that is applied on your gateway. Configuration might be missing on the gateway or might be applied to the wrong route. For example, if you apply multiple RouteOption resources to the same route by using the `targetRefs` section, only the oldest RouteOption is applied. The newer RouteOption configuration is ignored and not applied to the gateway. 
+3. If the resources seem to be ok, you can check the configuration that is applied on your gateway. Configuration might be missing on the gateway or might be applied to the wrong route. For example, if you apply multiple policies to the same route by using the `targetRefs` section, only the oldest policy is applied. The newer policy configuration might be ignored and not applied to the gateway. 
+   
    ```sh
-   {{< reuse "docs/snippets/cli-name.md" >}} get proxy -o yaml
+   kubectl describe pod -n {{< reuse "docs/snippets/ns-system.md" >}} <gateway-proxy-pod-name>
```

Comment:
so this is a little tricky I think. describing the pod won't show its config. I am wondering if we should have people look at the Gateway resource itself. or maybe check all policies, such as RouteOptions, VirtualHostOptions, etc.

Reply:
I think theoretically, we can remove this step, because in order to see the gateway config, they need to port-forward the pod which we suggest in the next step